coba/learners/vowpal.py

```diff
@@ -196,45 +196,39 @@ class VowpalLearner(Learner):
 
     @staticmethod
     def make_args(
-        options: Sequence[str],
-        noconstant: bool,
-        interactions: Sequence[str],
-        ignore_linear:Sequence[str],
-        seed: Optional[int],
-        **kwargs) -> str:
-        """Turn settings into a VW command line string.
+            vw_kwargs: Dict[str, Any],
+            namespace_interactions: Sequence[str] = DEFAULT_NAMESPACE_INTERACTIONS,
+    ) -> Sequence[str]:
+        """
+        Turn settings into VW command line arguments.
 
-        Args:
-            options: A sequence of string values that represent VW CLI options.
-            noconstant: Indicates if constant term should be included in the VW features.
-            interactions: A sequence of namespace interactions to use during learning.
-            ignore_linear: A sequence of linear namespaces to ignore during learning.
-            seed: A random number generator seed to make sure VW behaves consistently.
-            kwargs: Any number of additional options to add to the arg string.
+         Args:
+            vw_kwargs: Keyword argument dict that gets translated into VW CLI arguments
+            namespace_interactions: A sequence of namespace interactions to use during learning.
         """
+        args = []
+        vw_kwargs['quiet'] = vw_kwargs.get('quiet', True)
+
+        for k,v in vw_kwargs.items():
+            if v is not False and v is not None:
```

Comment:
This condition didn't include `v is not None` but I think for only `"flag_1"` to pass is the more expected behavior
```
{
    "flag_1": True,
    "flag_2": False,
    "flag_3": None
}
```